prediction/src/algorithms/evaluation.py

```diff
@@ -0,0 +1,58 @@
+import os
+import sys
+
+sys.path.insert(0, os.path.join(os.path.dirname(os.path.realpath(__file__)), '..', '..'))
+sys.path.append('..')
+import numpy as np
+import pylidc as pl
+from config import Config
+from classify import trained_model
+from tqdm import tqdm
+from src.preprocess.lung_segmentation import get_z_range
+
+CONFIDENCE_THRESHOLD = 0.5
+
+
+def get_accuracy(TP, TN, FP, FN):
+    """
+    The accuracy is the ability to correctly predict the class of an observation.
+    """
+    return (TP + TN) / (TP + TN + FP + FN)
+
+
+def logloss(true_label, predicted, eps=1e-15):
+    """Calculate the logarithmic loss (http://wiki.fast.ai/index.php/Log_Loss)"""
+    p = np.clip(predicted, eps, 1 - eps)
+    if true_label == 1:
+        return -np.log(p)
+    else:
+        return -np.log(1 - p)
+
+
+def evaluate_classification(model_path=None):
```

Comment:
Will it be better to make evaluation functionality console executable with appropriate metrics and directories arguments? Mean, that this is a bit out of user usage. 
I'd also suggest to put all metrics in a different file.